frontend/pages/redeem/thank-you.tsx

```diff
@@ -8,18 +8,23 @@ import CampaignIcon from '../../components/icons/CampaignIcon';
 import { useRouter } from 'next/router';
 import ContributeIcon from '../../components/icons/ContributeIcon';
 import ExploreIcon from '../../components/icons/ExploreIcon';
+import { isInteger } from 'formik';
 
 const ThankYou: NextPage = () => {
   const router = useRouter();
+  const campaignId = router.query.campaignId;
 
   const buttonProps = [
```

Comment:
Can we realign the buttons if there are only 2 of them?

These are some ways to do it, either one of them should work
1. Make each button have a defined width, then put the buttons in a Stack, direction="row", justifyContent="space-between", spacing={1} or something
2. Make the Grid item xs={12/buttonProps.length} <-- fractions should be allowed btw, I wrote {12/5} somewhere

<img width="980" alt="Screenshot 2022-10-24 at 5 55 35 PM" src="https://user-images.githubusercontent.com/71375383/197500723-97e300fa-02a6-47c7-a69b-3c79fa32649a.png">